src/XR/features/WebXRLayers.ts

```diff
@@ -69,7 +69,10 @@ class WebXRCompositionLayerRenderTargetTextureProvider extends WebXRLayerRenderT
         }
 
         this._lastSubImages.set(eye, subImage);
-
+        // don't skip clear for the first rtt. skip for all others
+        if (subImageIndex === 0) {
```

Comment:
nit: Does the WebXR driver not already clear the texture during the call to `getSubImage`?
https://immersive-web.github.io/layers/#xropaquetextures

Reply:
i'll test it again and write back here. If i remember correctly without that only one eye was rendering